language/guidelines/user-experience/performance/index.md

```diff
@@ -0,0 +1,63 @@
+---
+type: guideline
+title: Performance
+resources:
+  links:
+    - name: "North: Performance"
+      source: http://pointnorth.io/#performance
+    - name: "Have Your Cake and Eat it Too: A Primer on Performance"
+      source: http://una.im/slides-nom-nom-perf/#/
+    - name: "Our need for web speed: It’s about neuroscience, not entitlement"
+      source: http://www.webperformancetoday.com/2012/03/21/neuroscience-page-speed-web-performance/
+    - name: "Powers of 10: Time Scales in User Experience"
+      source: http://www.nngroup.com/articles/powers-of-10-time-scales-in-ux/
+    - name: "Web Page Test"
+      source: http://www.webpagetest.org/
+---
+
+Performance is an integral part of a user's experience no matter the medium that interface takes. Web delivered interfaces, native interfaces, application programing interfaces; whenever a user interacts with a product, performance always needs to be a top concern. Slow performing interfaces have been shown to [cost business money](http://www.fastcompany.com/1825005/how-one-second-could-cost-amazon-16-billion-sales), [affecting a company's bottom line](https://blog.kissmetrics.com/loading-time/) and costing conversions, but more than that, performance is about respecting our users and the way humans perceive and react to the world around them.
```

Comment:
application programming

cost money and conversions, affect a company's bottom line. More than that, though, performance is about respecting...